practice/src/assignment1/Main.kt

```diff
@@ -0,0 +1,32 @@
+package assignment1
+
+import java.util.*
+
+fun main() {
+
+    val sc = Scanner(System.`in`)
+    val N = sc.nextInt()
+
+    for (i in 1..N) {
```

Comment:
- 아마 N번 동작하는 것이 아니라 N - 1번 동작할 것 같습니다.
```suggestion
    repeat(N) {
```

- 이런식으로 바꾸실 수도 있습니다!